cmd/world/root/root.go

```diff
@@ -44,6 +52,63 @@ func init() {
 	logger.AddLogFlag(doctorCmd)
 }
 
+func checkLatestVersion() error {
+	url := "https://api.github.com/repos/Argus-Labs/world-cli/releases/latest"
+
+	// Create a request with the context
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, nil)
```

Comment:
nit: `http.Get` combines the NewRequestWithContext and Do methods.

Reply:
I've used it before, but the linter doesn't allow it.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/YO1Dcg4NByYdZHvKXaTq/fcc3e617-81d7-4bac-a332-ba8c39551614.png)